### PR TITLE
Truncate description in package list

### PR DESCRIPTION
**Pull request by msabramo**
Before:

screen shot 2017-03-09 at 12 04 42 pm

After:

screen shot 2017-03-09 at 1 24 06 pm

Cc: @tamarrow, @ssk2

### CODE REVIEW

File: cli/dcoscli/tables.py [hunk 1]
```diff
@@ -765,7 +765,7 @@ def package_table(packages):
          lambda p: '\n'.join(p['apps']) if p.get('apps') else EMPTY_ENTRY),
         ('COMMAND',
          lambda p: p['command']['name'] if 'command' in p else EMPTY_ENTRY),
-        ('DESCRIPTION', lambda p: p['description'])
+        ('DESCRIPTION', lambda p: textwrap.fill(p['description']))
```
Comment (tamarrow) [inline cli/dcoscli/tables.py]: lets use `truncate_table` here

**Comment by msabramo** [comment]
Switched to `truncate_table` in 8778726.

**Comment by tamarrow** [comment]
@msabramo can you rebase please? (tests updated for interface change)

**Comment by msabramo** [comment]
Rebased.

**Comment by tamarrow** [comment]
ok to test

**Pull request merged**